### PR TITLE
Add Boskos for build cluster

### PR DESCRIPTION
**Pull request by chaodaiG**
/assign chizhg

Part of: #1695

### CODE REVIEW

**Comment by chaodaiG** [comment]
@chizhg, please don't apply the build cluster yaml yet, otherwise it'll go ahead and clean up Boskos project 100

File: config/build-cluster/Makefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build cluster configuration
+# So far only the following usages work:
+# - update-all-boskos-deployments
+# - update-boskos-resource
+
+PROJECT                   := knative-tests
+CLUSTER                   := knative-prow-build-cluster
```
Comment (chizhg) [inline config/build-cluster/Makefile:21]: Remove the extra spaces?

File: config/build-cluster/cluster/400-boskos-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 The Knative Authors
```
Comment (chizhg) [inline config/build-cluster/cluster/400-boskos-deployment.yaml:1]:
```suggestion
# Copyright 2020 The Knative Authors
```

Comment (chaodaiG) [reply config/build-cluster/cluster/400-boskos-deployment.yaml:1]: done

**Pull request merged**